items/SPOOKY_PIE.json

```diff
@@ -8,9 +8,9 @@
     "§6Spooky Festival §7during",
     "§7SkyBlock Year 72.",
     "",
-    "§7Player: §6IRONM00N",
-    "§7Position: §a#4,353",
-    "§7Candy Score: §a651",
+    "§7Player: §6§kIRONM00N",
```

Comment:
😠This is my easter egg, Moulberry said I could have it

Reply:
Well your name is still there and the other items were formatted like this so